e2e/terraform/config/dev-cluster/nomad/client-linux/client.hcl

```diff
@@ -36,6 +36,9 @@ plugin "nomad-driver-ecs" {
 }
 
 vault {
-  enabled = true
-  address = "http://active.vault.service.consul:8200"
+  enabled   = true
+  address   = "https://active.vault.service.consul:8200"
+  ca_file   = "/etc/vault.d/tls/ca.crt"
+  cert_file = "/etc/vault.d/tls/agent.crt"
+  key_file  = "/etc/vault.d/tls/agent.key"
```

Comment:
The TLS configuration for vault needs to be set based on the value of the `tls` variable, I think? I think we should pull the vault block out into a shared `vault.hcl` file that gets overwritten by the `bootstrap-vault.sh` script.

Reply:
Oh yeah!  Good call out.  I'll circle back.

Reply:
What I have done as of now was add a Vault stanza to `shared/nomad-tls.hcl`. I noticed that that file already existed, was driven of the `tls` variable, and was setting Consul tls info. I reverted the address in `client.hcl` to its original state.